src/web/views/children-dob.njk

```diff
@@ -13,38 +13,17 @@
     }
   }) %}
 
-    {% call govukFieldset({
-      legend: {
-        text: aboutYourChild,
-        classes: "govuk-fieldset__legend--m"
-      }
-    }) %}
-
-    {{ govukInput({
-      label: {
-        text: nameLabel
-      },
-      id: 'children-name',
-      name: 'childrenName',
-      value: claim.childrenName,
-      errorMessage: errors | getErrorForField('childrenName'),
-      autocomplete: "off"
-    }) }}
-
-    {% endcall %}
-
-    {{ htbhfDateInput({
-      fieldNamePrefix: "childrenDob",
-      claim: claim,
+  {%  for i in range(0, children.count) %}
+    {{ htbhfChildDobInput({
+      index: i + 1,
+      children: children,
```

Comment:
I wonder if the input fields need to be `child` rather than `children`, `childrenDob-01` for example doesn't sound quite right, but `childDob-01` seems better.

Reply:
Wouldn't it make sense to pass in the child (`children[i]`) to htbhfChildDobInput, rather than `children`?

Reply:
Ignore me, I was forgetting what `children` was in this context